connector/oidc/oidc.go

```diff
@@ -351,6 +356,7 @@ func (c *oidcConnector) createIdentity(ctx context.Context, identity connector.I
 		if err := userInfo.Claims(&claims); err != nil {
 			return identity, fmt.Errorf("oidc: failed to decode userinfo claims: %v", err)
 		}
+		c.logger.Debugf("UserInfo::: %q \n", userInfo)
```

Comment:
I assume this stayed here by accident?

Reply:
Similar to  the previous need, it was helpful to see to which groups the user belonged to. Again, I will remove it if you think it is not necessary.

Reply:
Removed this in 345eba905ddb2664855e79da2acf816ad6b97eb5